src/dbus.c

```diff
@@ -324,12 +324,20 @@ static void on_get_server_information(GDBusConnection *connection,
 
 void notification_closed(notification *n, enum reason reason)
 {
+        int valid_reason = REASON_UNDEF;
+
+        if (REASON_INVALID_MIN < reason && reason < REASON_INVALID_MAX) {
+                valid_reason = reason;
```

Comment:
Any reason for using a different variable here? Why not just `reason = REASON_UNDEF`?

Reply:
I actually dislike it, changing invocation parameters. But your objection is correct. I've changed it.